work16/index.js

```diff
@@ -0,0 +1,248 @@
+
+const REQUEST_URL = "https://mocki.io/v1/21ee587f-5a59-40ff-98c6-46b2d4715f25"
+
+// 間違っているURLの場合↓
+// const REQUEST_URL = "https://myjson.dit.upm.es/api/bins/ほげほげajy3";
+// 空の配列の場合↓
+// const REQUEST_URL = "https://mocki.io/v1/242b685f-a3d7-45a8-aeca-0376bd495b89";
+// 503エラーの場合↓
+// const REQUEST_URL = "https://httpstat.us/503";
+
+const request = async () => {
+    const response = await fetch(REQUEST_URL, {
+        headers: {
+            "Content-Type": "application/json"
+        }
```

Comment:
Q.I'm afraid I haven't studied enough.
What is the reason for specifying Content-type here?
My understanding is that it indicates the format of data sent to the server by POST method, etc.